datadog_checks_dev/datadog_checks/dev/tooling/commands/release/trello/testable.py

```diff
@@ -174,7 +181,10 @@ def pick_card_member(config: dict, author: str, team: str, card_assignments: dic
         random.shuffle(team_members)
         card_assignments[team] = dict.fromkeys(team_members, 0)
 
-    member = min([member for member in card_assignments[team] if member != author], key=card_assignments[team].get)
+    member = min(
+        [member for member in card_assignments[team] if member != author and member not in approvers],
+        key=card_assignments[team].get,
+    )
```

Comment:
Note that there's an edge case here where all team members have approved a PR (unlikely, I know) where this would raise an exception.

Reply:
This would also be the case if a team has only one member

Reply:
Should be good now! If this happens, it will be handled by another function https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/commands/release/trello/testable.py#L46

Let me know what you think